language/vm/src/file_format.rs

```diff
@@ -385,6 +385,8 @@ pub enum Visibility {
     Private = 0x0,
     /// Accessible by any module or script outside of its declaring module.
     Public = 0x1,
+    /// Accessible by any script or other `Script` functions from any module
+    Script = 0x3,
```

Comment:
Is this 0x3 because 0x2 is reserved for friend?

Reply:
It's because I mistyped :P